connectors/spark/src/main/scala/io/unitycatalog/connectors/spark/UCSingleCatalog.scala

```diff
@@ -6,6 +6,7 @@ import io.unitycatalog.client.model.{ColumnInfo, ColumnTypeName, CreateSchema, C
 
```

Comment:
why we have this PR, can we change the package names to be shorter `io.unitycatalog.spark`